File: Python/26_RemoveDuplicatesFromSortedArray.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if len(nums) == 0:
            return 0

        count = 1
        i = 1          
        while i < len(nums):
            if (nums[count-1] == nums[i]):
                for n in range(i+1, len(nums)):
                    if nums[count-1] != nums[n]:
                        nums[count] = nums[n]
                        i = n - 1
                        count  += 1
                        break
            else:
                count  += 1
            i += 1
                
        return count

File: C#/26_RemoveDuplicatesFromSortedArray.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        if (nums.Length == 0) return 0;
        if (nums.Length == 1) return 1;
        int curr = 0;
        for (int i = 1; i < nums.Length; i++){
            if (nums[i] != nums[curr]){
                curr++;
                nums[curr] = nums[i];
            }
        }
        return curr + 1;
    }
}
